src/daft-image/src/series.rs

```diff
@@ -78,10 +76,9 @@ pub fn decode(
 ) -> DaftResult<Series> {
     match s.data_type() {
         DataType::Binary => image_decode_impl(s.binary()?, raise_error_on_failure, mode)
-            .map(|arr| arr.into_series()),
+            .map(daft_core::series::IntoSeries::into_series),
```

Comment:
also don't really like this rule, the former is imo easier to read, and modify if needed.

Reply:
I'm okay with dropping this lint if we want. I don't think it really adds much value.

Reply:
696f1f18173a0aa3f126da67aa61cc560ecd389b